### ISSUE TITLE
Incompatible with `Snakemake>=8`

### ISSUE THREAD

**Issue by votti** [gagneurlab/drop#534]
Currently `drop` seems incompatible with the most recent `Snakemake` version.

## Reproducible issue
- Install `drop`: `mamba create -n drop_env_debug -c conda-forge -c bioconda drop --override-channels`
- Move to a drop envrionment (eg the demo one)
- Run `snakemake -n` or any other `snakemake` command

Error:
```
:~/drop_demo$ snakemake -n
ImportError in file /home/vitoz/drop_demo/Snakefile, line 3:
cannot import name 'get_argument_parser' from 'snakemake' (/home/vitoz/mambaforge/envs/drop_env_debug/lib/python3.12/site-packages/snakemake/__init__.py)
  File "/home/vitoz/drop_demo/Snakefile", line 3, in <module>
  File "/home/vitoz/mambaforge/envs/drop_env_debug/lib/python3.12/site-packages/drop/__init__.py", line 1, in <module>
  File "/home/vitoz/mambaforge/envs/drop_env_debug/lib/python3.12/site-packages/drop/setupDrop.py", line 2, in <module>
  File "/home/vitoz/mambaforge/envs/drop_env_debug/lib/python3.12/site-packages/drop/config/__init__.py", line 1, in <module>
  File "/home/vitoz/mambaforge/envs/drop_env_debug/lib/python3.12/site-packages/drop/config/DropConfig.py", line 1, in <module>
  File "/home/vitoz/mambaforge/envs/drop_env_debug/lib/python3.12/site-packages/drop/config/SampleAnnotation.py", line 1, in <module>
  File "/home/vitoz/mambaforge/envs/drop_env_debug/lib/python3.12/site-packages/drop/utils.py", line 3, in <module>
  File "/home/vitoz/mambaforge/envs/drop_env_debug/lib/python3.12/site-packages/wbuild/__init__.py", line 10, in <module>
  File "/home/vitoz/mambaforge/envs/drop_env_debug/lib/python3.12/site-packages/wbuild/autolink.py", line 6, in <module>
  File "/home/vitoz/mambaforge/envs/drop_env_debug/lib/python3.12/site-packages/wbuild/utils.py", line 12, in <module>
```

## Workaround
Restrict `snakemake` version during installation:
`mamba create -n drop_env -c conda-forge -c bioconda drop snakemake-minimal=7.32 --override-channels`

## Mitigation
- Change example installation command.
- Specify dependency in `drop` conda release
- This seems to be a `wbuild` issue and I will open add an issue there as well.

**Comment by vyepez88** [comment]
many thanks! we're fixing this and let you know once it's done

**Comment by vyepez88** [comment]
Hi @votti, many thanks for your help! Really appreciate it. This should be now fixed in v 1.4.0
